testing/python_lib/integration_base.py

```diff
@@ -141,6 +145,44 @@ def _get_faucet_config_path(self):
         return os.path.dirname(os.path.abspath(__file__)) + \
             (config_file_format % ('forch-faucet-1'))
 
+    def parallelize(self, job_count, target, target_args=None):
+        """Parallelizes multiple runs of a target method with multiprocessing"""
+        jobs = []
+        if not target_args:
+            target_args = []
+        for job in range(job_count):
+            if job < len(target_args):
```

Comment:
is this complexity even necessary?  Seems like in all cases you just need the target_args and target, the "job_count" field is redundant?  Seems like this method could be simplified by removing the "job_count" argument and always having job_count=len(target_args)

Reply:
But what if the method has no arguments. Having to push an array with a set of empty tuples seems wrong.

Reply:
Changed.